src/App.jsx

```diff
@@ -14,70 +16,66 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [thisGood, setGood] = useState('Jam');
```

Comment:
```suggestion
  const [thisGood, setThisGood] = useState('Jam');
```
Destructured params should correspond